srcpkgs/common-lisp-ppcre/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'common-lisp-ppcre'
+pkgname=common-lisp-ppcre
+version=2.1.1
+revision=1
+hostmakedepends="curl"
+short_desc="Common Lisp regular expression library"
+maintainer="ctoid <funk443@yandex.com>"
+license="BSD-2-Clause"
+homepage="https://edicl.github.io/cl-ppcre/"
+distfiles="https://github.com/edicl/cl-ppcre/archive/refs/tags/v${version}.tar.gz"
+checksum="89631179b71648d9e6c565a928f6896a9d5742aa2083b9c1b705fe0b45d85def"
+
+_LIBRARY_DIR=/usr/share/common-lisp/source/cl-ppcre
+
+do_install() {
+	curl -O "https://raw.githubusercontent.com/edicl/cl-ppcre/master/LICENSE"
```

Comment:
Please extract license from comment, or move to distfiles with a note to use copy from tarball on nearest update.